iron-collapse.html

```diff
@@ -142,7 +143,7 @@
     },
 
     updateSize: function(size, animated) {
-      this.enableTransition(animated);
+      this._updateTransition(animated);
```

Comment:
just did it, it makes the code cleaner too! (got rid of `ready` like this :+1: )